### DISCUSSION TITLE
Dataview query not rendered?

### DISCUSSION THREAD

**Discussion by dlleigh** [brianpetro/obsidian-smart-connections#781] [General]
Following the [option (b) suggestion in this thread](https://github.com/brianpetro/obsidian-smart-connections/discussions/509#discussioncomment-8987487), I'm using a dataviewjs block to aggregate the content of several notes and asking smart chat to summarize the note containing the dataview.  However instead of using the rendered query, smart connections is summarizing the dataview block and giving responses like:

```
This note appears to be a code snippet written in DataviewJS, which is used for filtering and displaying information in the Dataview app.
```

Any suggestions?

**Comment by brianpetro** [comment]
@dlleigh this may be a bug with the rendering dataviews. 

It might be specific to dataviewjs code-blocks though.

Can you test if it works with a non-js dataview code-block?

🌴

**Comment by dlleigh** [comment]
Yes, I can confirm that it works with a non-js dataview code block.
